src/scheduler/workers/unpublish/weekly/types.ts

```diff
@@ -0,0 +1,20 @@
+import { Address, List, ListItem } from "@prisma/client";
+
+export interface Meta {
+  /**
+   * annual review reference for list (`List.jsonData.currentAnnualReview.reference`)
+   */
+  reference: string;
+
+  /**
+   * Weeks until unpublish. Used for debugging. added to `Event.jsonData.notes`.
+   */
```

Comment:
I'm not sure if these comments are needed 🤔

Reply:
done -- but I've left the comment around where `reference` comes from since there can be confusion between listItem.reference and list.jsonData.currentAnnualReview.reference